backend/tester/project/Dependencies.scala

```diff
@@ -0,0 +1,35 @@
+import sbt._
+
+object Dependencies {
+
+  val calibanVersion = "1.3.1"
+  val sttpVersion = "3.3.18"
+  val zioVersion = "1.0.13"
```

Comment:
Caliban over Sangria, Sttp over Http4s, and Zio over Cats Effect? 😄

Reply:
Caliban because this client gen support is Caliban only
Sttp is a client lib, http4s is a server lib (we are using both here)
ZIO because Caliban is a ZIO project :)